pkg/admin/service.go

```diff
@@ -1,28 +1,34 @@
 package admin
 
-import "context"
+import (
+	"context"
+	"github.com/pyroscope-io/pyroscope/pkg/storage"
+)
 
 type AdminService struct {
 	storage Storage
 }
 
 type Storage interface {
-	GetAppNames(context.Context) []string
-	DeleteApp(ctx context.Context, appname string) error
+	storage.AppGetter
+	storage.AppDeleter
 }
 
 func NewService(v Storage) *AdminService {
 	m := &AdminService{
 		v,
 	}
-
 	return m
 }
 
-func (m *AdminService) GetApps() (appNames []string) {
-	return m.storage.GetAppNames(context.TODO())
+func (m *AdminService) GetAppNames(ctx context.Context) (appNames []string) {
```

Comment:
It seems the service does nothing but proxying requests to the storage. Please consider removing it